combat-trainer.lic

```diff
@@ -1425,15 +1426,55 @@ class AbilityProcess
     @barb_buffs = @buffs.delete('barb_buffs') || []
     echo("  @barb_buffs: #{@barb_buffs}") if $debug_mode_ct
     setup_barb_buff_flags
+
+    @paladin_use_badge = settings.paladin_use_badge
+    echo("  @kneel_khri: #{@kneel_khri}") if $debug_mode_ct
```

Comment:
Should this be `@paladin_use_badge`?

Reply:
Yeah, caught it in my current iteration already.